_data/simple-icons.json

```
@@ -9790,6 +9790,19 @@
             "hex": "00BCF6",
             "source": "https://www.rtl2.de/"
         },
+        {
+            "title": "RTM",
+            "hex": "244FA2",
```

```suggestion
            "hex": "36474F",
```
Picked color from the SVG logo on their website https://www.rtm.fr